extensions/amp-list/0.1/amp-list.js

```diff
@@ -64,27 +67,54 @@ export class AmpList extends AMP.BaseElement {
 
   /** @override */
   layoutCallback() {
-    return this.populateList_();
+    const populate = this.populateList_();
+    populate.catch(error => {
+      // Normally firstLayoutCallback() hides the placeholder, but it won't
+      // get called if this promise is rejected.
+      this.togglePlaceholder(false);
```

Comment:
Based on conversations here: https://github.com/ampproject/amphtml/pull/10281 and to stay consistent with other elements (also this would introduce a breaking change in behaviour) I think we should keep displaying the placeholder unless there is a fallback specified when failure happens.